app/models.js

```diff
@@ -47,9 +47,31 @@ function fetchCommentsByArticleId(article_id) {
     });
 }
 
+function insertComment(newComment, article_id) {
+  const { username: author, body } = newComment;
+
+  if (
+    (!author || !body || typeof author !== "string", typeof body !== "string")
```

Comment:
PSQL should be triggering an error if any of these happen i.e. a null value, a type of data that doesn't match the column. This could be handled just by your error handler